webapp/publisher/snaps/views.py

```diff
@@ -1356,11 +1357,29 @@ def get_snap_builds(snap_name):
     }
 
     # Get built snap in launchpad with this store name
+    github = GitHubAPI(flask.session.get("github_auth_secret"))
     lp_snap = launchpad.get_snap_by_store_name(details["snap_name"])
 
+    try:
+        context["github_user"] = github.get_username()
```

Comment:
If this snap has `builds` lower down, do we need any information at all from their GitHub account? 'cos if not, we should move this request lower down in the logic so we're not making it if we don't need to.

Reply:
You are right, I moved it at the top when trying to use GraphQL to check the yaml file in the repo. But now that method doesn't need user authentication.